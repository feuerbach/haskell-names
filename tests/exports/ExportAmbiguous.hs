module ExportAmbiguous (DataType(..)) where

import qualified Prelude (DataType(..))
import Prelude hiding (DataType(..))

data DataType = Constructor1 | Constructor2
