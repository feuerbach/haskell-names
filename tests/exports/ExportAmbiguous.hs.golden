[ Data
    { symbolModule = ModuleName () "ExportAmbiguous"
    , symbolName = Ident () "DataType"
    }
, Constructor
    { symbolModule = ModuleName () "ExportAmbiguous"
    , symbolName = Ident () "Constructor1"
    , typeName = Ident () "DataType"
    }
, Constructor
    { symbolModule = ModuleName () "ExportAmbiguous"
    , symbolName = Ident () "Constructor2"
    , typeName = Ident () "DataType"
    }
]